api_docs/markdowns/api/collections/collections.md

```diff
@@ -134,8 +133,8 @@ The endpoint response is paginated and in JSON format. It is formalised as such:
 First, a typical cURL pattern to use the collections API:
 
 ```bash
-curl --request POST \
-  --url https://app.botify.com/api/v1/projects/{{username}}/{{project_slug}}/query?size=50 \
+curl -X POST \
```

Comment:
Sorry I wasn't clear: neither `-X` not `--request` is right. It changes the HTTP verb without changing anything else; it's `--data` that makes cURL use the POST protocol. (Sources: https://curl.haxx.se/docs/faq.html#curl_X_gives_me_HTTP_problems, https://daniel.haxx.se/blog/ some time ago :slightly_smiling_face:)

Reply:
Awesome, thanks! Didn't know that